docker-entrypoint.sh

```diff
@@ -78,6 +78,20 @@ if [[ -n ${self_hostname} ]]; then
   sed -i "s/self_hostname: localhost/self_hostname: $self_hostname/g" "$CHIA_ROOT/config/config.yaml"
 fi
 
+if [[ -n ${full_node_peer} ]]; then
+  echo "Changing full_node_peer settings in config.yaml with value: $full_node_peer"
+  full_node_peer_host=$(echo "$full_node_peer" | cut -d ':' -f 1) \
+  full_node_peer_port=$(echo "$full_node_peer" | cut -d ':' -f 2) \
```

Comment:
I think this would probably break w/ ipv6 (maybe somewhat unlikely in docker at this point, but if we can support it, we should)

Reply:
Updated